kcidb/db/__init__.py

```diff
@@ -259,6 +260,30 @@ def complement(self, data):
         return self.query(ids=ids, children=True, parents=True)
 
 
+class DBHelpAction(argparse.Action):
+    """Argparse action outputting database string help and exiting."""
+    def __init__(self,
+                 option_strings,
+                 dest=argparse.SUPPRESS,
+                 default=argparse.SUPPRESS,
+                 help=None):
+        super().__init__(
+            option_strings=option_strings,
+            dest=dest,
+            default=default,
+            nargs=0,
+            help=help)
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        print("Specify database as <DRIVER>:<PARAMS>. " +
+              "Driver options and associated params (if any) " +
+              "are listed below.\n")
```

Comment:
This help message isn't really about the `-d`/`--database` option, but rather about its value, so we shouldn't start with "Specify database as <DRIVER>:<PARAMS>.", but rather with a general introduction, the fact that there are multiple databases supported, and the format of the specification string. So, something like this:

```suggestion
        print("KCIDB has several database drivers for both actual and "
              "virtual databases.\n" 
              "You can specify a particular driver to use, and its "
              "parameters, using the\n"
              "-d/--database option.\n"
              "\n"
              "The format of the option value is <DRIVER>[:<PARAMS>], "
              "where <DRIVER> is the\n"
              "name of the driver, and <PARAMS> is a (sometimes optional) "
              "driver-specific\n"
              "parameter string.\n"
              "\n"
              "For example, \"-d bigquery:kernelci-production.kcidb_01\" "
              "requests the use of\n"
              "the \"bigquery\" database driver with the parameter string\n"
              "\"kernelci-production.kcidb_01\", from which the driver "
              "extracts the Google\n"
              "Cloud project \"kernelci-production\" and the dataset "
              "\"kcidb_01\" to connect to.\n"
              "\n"
              "Available drivers and format of their parameter strings "
              "follow.\n")
```